src/platform/packages/private/kbn-ui-shared-deps-src/src/entry.js

```diff
@@ -8,6 +8,15 @@
  */
 
 require('./polyfills');
+// Optional prototype hardening. This must occur immediately after polyfills.
+if (typeof window.__kbnHardenPrototypes__ !== 'boolean') {
```

Comment:
This looks a bit strict - are we sure this will be set in every scenario? Tests/ext-plugins included?